uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -63,15 +61,22 @@
 public class MqttControlSteps {
     private static final String DEFAULT_CLIENT_DEVICE_POLICY_CONFIG = "/configs/iot/basic_client_device_policy.yaml";
 
+    private static final String CA_SEPARATOR = "\n\n\n";
```

Comment:
why 3 new lines? A single newline should be sufficient.

Reply:
I'm not sure all available MQTT v5.0 libraries are supporting joined PEM format. 
It not broke a PEM formatting but allow to split back to separate CA by only one String.split().
Single "\n" does not allow that due to each line ended by \n.

Reply:
CA should be provided to the MQTT client as a list of certificates. The MQTT client should then handle that list in an appropriate way, which in most cases will be to concatenate the certificates.

Reply:
Ok, will change the protocol, control and existing client.

Reply:
But that does not solve CA_SEPARATOR issue due to scenario context can store only strings, I must join all certificates to single string on processDiscoveryResponse() and split in connect().

Reply:
What does this have to do with the scenario context? Scenario context should not be involved here.

Store the discovery information as an object and provide it over gRPC when connecting the client.

Reply:
Everything in the step class is `@ScenarioScoped` which means it is recreated for each scenario execution. There must not be any conflict between scenario executions.

Reply:
Proto updated to move list of CA in b86831a4ce0a3fc0ee7f56f7f2e4b0188757e623. Implementations also changed.